shimmer/src/main/java/com/cooltechworks/views/shimmer/ShimmerRecyclerView.java

```diff
@@ -104,13 +109,15 @@ private void init(AttributeSet attrs) {
             mShimmerAngle = a.getInteger(R.styleable.ShimmerRecyclerView_shimmer_demo_angle, 0);
             mShimmerColor = a.getColor(R.styleable.ShimmerRecyclerView_shimmer_demo_shimmer_color, getColor(R.color.default_shimmer_color));
             mShimmerItemBackground = a.getDrawable(R.styleable.ShimmerRecyclerView_shimmer_demo_view_holder_item_background);
+            mShimmerDuration = a.getInteger(R.styleable.ShimmerRecyclerView_shimmer_demo_duration, 0);
```

Comment:
The default value is 0. So if one do not add duration attribute then the animation will play since the defautl duration is zero. It must be changed.

Reply:
I will set mShimmerDuration, if user gives explicitly. Since default value(1500) is already set in class ShimmerLayout. So will remove this lines and go with line 85.

Reply:
fixed.